notebooks/243-depth-estimation-videpth/243-depth-estimation-videpth.ipynb

```diff
@@ -0,0 +1,1088 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "98466560-036d-41d2-87f7-957af6f64626",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Monocular Visual-Inertial Depth Estimation using OpenVINO™\n",
+    "\n",
+    "<p align=\"center\" width=\"100%\">\n",
+    "    <img src=\"https://raw.githubusercontent.com/isl-org/VI-Depth/main/figures/methodology_diagram.png\">    \n",
```

Comment:
@pronoym99 Please ensure to provide appropriate credit for the diagram and images used.